dspace-api/src/main/java/org/dspace/authorize/ResourcePolicy.java

```diff
@@ -109,6 +108,37 @@ public boolean equals(Object obj)
     }
 
     /**
+     * Create a new resource policy based on this one but copy it to another DSpace object
```

Comment:
Please specify in the code comments why this is more efficient than just creating the record first and then setting all the field values (-> requires only insert statement vs  1 insert + 1 update).